### DISCUSSION TITLE
Possible to use it without blade component?

### DISCUSSION THREAD

**Discussion by aneesdev** [dietercoopman/smart#19] [Q&A] [answered]
Hi @dietercoopman this is great package you made and I really appreciate it. is it possible to use this package without x-smart blade component? i mean for example i use this package to get the resized image src and pass to my vue component etc.

Thanks

**Comment by dietercoopman** [comment, marked as answer]
Hi @aneesdev , at the moment it is not possible to use it with a vue component.  If you want to use the logic behind smart in your Vue backend you might take a look at the intervention/image package, that's the package smart uses for it's magic.

I might take a look at a Vue implementation later but it's not a priority at the moment 

thx for the question and have a nice day 😎

**Reply by aneesdev** [reply]
thanks for replying
